info/v1/container.go

```diff
@@ -516,6 +516,31 @@ type FsStats struct {
 	WeightedIoTime uint64 `json:"weighted_io_time"`
 }
 
+// NvidiaGpuStats are measured using NVML.
+type NvidiaGpuStats struct {
+	// Minor number of the GPU device (/dev/nvidia[minor_number])
+	MinorNumber uint64 `json:"minor_number"`
+
+	// Product name of the GPU
+	Name string `json:"name"`
+
+	// Total installed GPU memory.
+	// unit: bytes
+	MemoryTotal uint64 `json:"memory_total"`
```

Comment:
This should be a constant, right?  If so, does it belong in the monitoring pipeline?  Would it be better to surface it as a "spec" somewhere?

Reply:
It's constant for a particular GPU, but there can be multiple GPUs with different total memory each. If I move it to spec I would have to have a way to link the total memory of a GPU with its usage. I felt using two different time series while plotting would be inconvenient for the user.

Reply:
Ok, I think that is reasonable.